src/confcom/azext_confcom/README.md

```diff
@@ -0,0 +1,484 @@
+
+      
+
+
+## Microsoft Azure CLI 'confcom' Extension Examples 
+
+Run `az confcom acipolicygen --help` to see a list of supported arguments along with explanations. The following commands demonstrate the usage of different arguments to generate confidential computing security policies. 
+
+**Note:** The Azure Confidential Computing CLI extension is in public preview and is subject to change. Some arguments may be added or removed and the way `confcom acipolicygen` command is called to achieve specific functionality may change as well. This documentation will be updated as changes to the tooling are published. 
+
+**Prerequisites:**
+Install the Azure CLI and Confidential Computing extension. 
+
+See the most recently released version of `confcom` extension.
+
+    az extension list-available -o table | grep confcom 
+
+To add the most recent confcom extension, run:
+
+    az extension add --name confcom
+
+Use the `--version` argument to specify a version to add. 
+ 
+You can also add the extension from a URL:
+
+    az extension add --source https://acccliazext.blob.core.windows.net/confcom/confcom-0.2.8-py3-none-any.whl -y
+
+
+The `acipolicygen` command generates confidential computing security policies using an image, an input JSON file, or an ARM template. You can control the format of the generated policies using arguments. Note: It is recommended to use images with specific tags instead of the `latest` tag, as the `latest` tag can change at any time and images with different configurations may also have the latest tag.
+
+**Examples:**
+
+Example 1: The following command creates a CCE policy and outputs it to the command line: <br /> 
+
+    az confcom acipolicygen -a .\template.json --print-policy
+
+This command combines the information of images from the ARM template with other information such as mount, environment variables and commands from the ARM template to create a CCE policy. 
+The `--print-policy` argument is included to display the policy on the command line rather than injecting it into the input ARM template.
+
+Example 2: This command injects a CCE policy into [ARM-template](arm.template.md) based on input from [parameters-file](template.parameters.md) so that there is no need to change the ARM template to pass variables into the CCE policy: <br />
+
+    az confcom acipolicygen -a .\arm-template.json -p .\template.parameters.json
+
+This is mainly for decoupling purposes so that an ARM template can remain the same and evolving variables can go into a different file. 
+
+Example 3: This command takes the input of an ARM template to create a human-readable CCE policy in pretty print JSON format and output the result to the console. 
+NOTE: Generating JSON policy is for use by the customer only, and is not used by ACI In most cases. The default REGO format security policy is required. <br />
+
+    az confcom acipolicygen -a ".\arm_template" --outraw-pretty-print --json --print-policy
```

Comment:
the behavior changed so only `--outraw-pretty-print` is needed. `--print-policy` should be omitted in this case

Reply:
I remember if we omit print policy, it won't print. Was there a change in CLI?

Reply:
yeah, it changed in version 0.2.9 so it's very new